lib/commands/audit.js

```diff
@@ -4,7 +4,7 @@ const fetch = require('npm-registry-fetch')
 const localeCompare = require('@isaacs/string-locale-compare')('en')
 const npa = require('npm-package-arg')
 const pacote = require('pacote')
-const { PromisePool } = require('@supercharge/promise-pool')
+const pMap = require('p-map')
```

Comment:
i strongly discourage relying on this package, since its author has committed to making all their packages ESM-only.

Reply:
@ljharb good point, I specifically depended on version 4.0.0 here, which doesn't use esm, but we might get stuck on that version for a very long time.

@wraithgar @darcyclarke thoughts on this?

Reply:
ops missed your replies @wraithgar 👍